lib/right_api_client/client.rb

```diff
@@ -369,7 +369,11 @@ def do_get(path, params={})
       data = if resource_type == 'text'
         { 'text' => body }
       else
-        JSON.parse(body, :allow_nan => true)
+        if res && res.headers[:content_type].split(';').first.strip == 'text/plain'
+          {:value => body}
```

Comment:
just a question on style:  spaces between the text and brackets?  Other hashes have them in this file.